src/block_service/fds/fds_service.cpp

```diff
@@ -681,36 +651,10 @@ error_code fds_file_object::put_content(/*in-out*/ std::istream &is,
         return err;
     }
 
-    try {
-        // Get Object meta data
-        std::shared_ptr<galaxy::fds::FDSObjectMetadata> metadata =
-            c->getObjectMetadata(_service->get_bucket_name(), _fds_path);
-        const std::map<std::string, std::string> metaMap = metadata->metadata();
-
-        auto iter = metaMap.find(fds_service::FILE_MD5_KEY);
-        dassert(iter != metaMap.end(),
-                "can't find %s in object(%s)'s metadata",
-                fds_service::FILE_MD5_KEY.c_str(),
-                _fds_path.c_str());
-        _md5sum = iter->second;
-
-        // in a head http-request, the file length is in the x-xiaomi-meta-content-length
-        // while in a get http-request, the file length is in the contentLength
-        iter = metaMap.find(fds_service::FILE_LENGTH_CUSTOM_KEY);
-        dassert(iter != metaMap.end(),
-                "can't find %s in object(%s) metadata",
-                fds_service::FILE_LENGTH_CUSTOM_KEY.c_str(),
-                _fds_path.c_str());
-        _size = (uint64_t)atoll(iter->second.c_str());
```

Comment:
`get_file_meta` will update `_has_meta_synced = true`, is it safe to add this sentence here?

Reply:
If we coule get object meta and update `_md5sum` and `_size`, setting `_has_meta_synced` true should be resonable.